pkg/driver/docker/docker.go

```diff
@@ -90,9 +90,27 @@ func (d *dockerDriver) StartDevContainer(ctx context.Context, id string, labels
 }
 
 func (d *dockerDriver) DeleteDevContainer(ctx context.Context, id string, deleteVolumes bool) error {
-	// TODO: implement deleteVolumes
+	var volume string
 
-	return d.Docker.Remove(ctx, id)
+	if deleteVolumes {
+		// inspect container deleteVolumes
+		container, err := d.Docker.InspectContainers([]string{id})
+		if err != nil {
+			return err
+		}
+		volume = container[0].Config.Labels["dev.containers.id"]
+	}
+
+	err := d.Docker.Remove(ctx, id)
+	if err != nil {
```

Comment:
Personal preference, so feel free to ignore.

```suggestion
	if err := d.Docker.Remove(ctx, id); err != nil {
```